package.json

```diff
@@ -38,7 +38,8 @@
     "rimraf": "^3.0.2",
     "rxjs": "^7.2.0",
     "swagger-ui-express": "^4.3.0",
-    "typeorm": "^0.2.44"
+    "typeorm": "^0.2.44",
+    "yarn": "^1.22.18"
```

Comment:
@fernandogcabal @jfernandezdev14 We use Yarn in our local enviroments as tool that allow us to manage the project, execute it, execute its scripts, its tests etc... but we don't use Yarn anywhere in the code, Yarn's something we use from the `outside`  that's why it is installed as a npm package BUT as a global package included in your pc's console but not in the project itself, because in reality, does not use yarn for anything, we do. I hope this is not very confusing, if you'd like we can discuss it a bit deeper in one of our meetings.